src/index/formatFileStructure/findEntryPoints.ts

```diff
@@ -1,53 +1,44 @@
 import { Graph } from "./shared/Graph";
 import { isTestFile } from "./shared/isTestFile";
 
-const invertGraph = (graph: Graph) => {
+function createInvertedGraph(graph: Graph) {
   const invertedGraph: Graph = {};
-  Object.keys(graph).forEach(k => {
-    graph[k].forEach(v => {
-      if (!(v in invertedGraph)) {
-        invertedGraph[v] = [];
-      }
-
-      invertedGraph[v].push(k);
-    });
-  });
-
+  const starts = Object.keys(graph);
+
+  for (const start of starts) {
```

Comment:
Changing to for of loop again. If anything this should be changed to a `forEach` loop over `Object.entries(graph)`.

Reply:
You're right, great find!